autoscheduler/frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import * as styles from './Schedule.css';
+import Meeting from '../../types/Meeting';
+import MeetingCard from '../MeetingCard/MeetingCard';
+
+interface ScheduleProps extends RouteComponentProps {
+  schedule: Meeting[];
+}
+
+const Schedule: React.FC<ScheduleProps> = ({ schedule }) => {
+  // these must be unique because of how they're used below
+  const DAYS_OF_WEEK = ['M', 'T', 'W', 'R', 'F'];
+  const FIRST_HOUR = 8;
+  const LAST_HOUR = 21;
+
+  // helper function for formatting
+  function formatHours(hours: number): number {
+    return ((hours - 1) % 12) + 1;
+  }
+
+  // values computed from props
```

Comment:
This is pretty nitpicky, but this comment (line 22) isn't very helpful, and I think the variable name of `uniqueSections` is pretty descriptive, so I don't think it's needed. 

Unless you mean that like all of the values below line 22 are values computed from props (rather than just the line below), then I think it should be like `/* values computed from props */` as I think the multi-line comments are better for logical "block" separation.

Reply:
I was going for the block separation